src/components/Main.vue

```diff
@@ -336,6 +336,7 @@ export default {
     changeCurrentUser(id) {
       const index = this.users.findIndex(user => user.id === id);
       this.setCurrentUser(this.users[index]);
```

Comment:
@Tyncture just FYI after changing the graph data to be a computed property it was longer being updated when a user was selected from the 'profiles' section so I added that function call here.